README.md

```diff
@@ -2,53 +2,76 @@
-LoopBack makes it easy to build modern applications that require complex integrations. LoopBack.next is the next version of LoopBack and is being designed to make it even easier to extend the framework for your own needs and will enable the latest ES features.
+## Table of Contents
+* [Overview](#overview)
+* [Setup](#setup)
+* [Testing](#testing)
+* [Project Structure](#structure)
+* [TypeScript Configuration](#typescriptConfiguration)
+* [Dependencies](#dependencies)
+* [Other Resources](#resources)
+* [Contribute](#contribute)
 
-Goal of tutorial
-Show you what a LoopBack.next app looks like and how the basics works
+## <a name="overview"></a>Overview
 
-what we're trying to do
-start a server to serve content
-make a request to the server to fetch content
+#### Welcome to the quick start tutorial for Loopback.next + TypeScript!
+Loopback makes it easy to build modern applications that require complex integrations. 
 
-how we do it
-create a server
-add a custom component
-add a custom sequence
+Loopback.next is the next version of LoopBack and is being designed to make it even easier to extend the framework for your own needs and will enable the latest ES features.
 
-What we'll do to the app:
-- Make a request to the server with a name and get back a HTTP 200 response
+The goal of this tutorial is to show you what a basic, 'hello-world' Loopback.next app looks like and how it works.
 
-## Setup
+This example will show you how to: 
 
-[CLEANUP]
-Before you clone ... 
-### Install `node`, `npm`, and `TypeScript` 
-You should have `node` (>= 7.6.0), `npm` (>= 3.10), and `TypeScript`  (>= 2.0.0) installed. For the latest information on project dependencies for LoopBack, please check this link: 
-https://github.com/strongloop/loopback-next/wiki/Installation
+* create a Loopback.next server
+* make a request to the server to fetch content
 
-```
-git clone https://github.com/strongloop/loopback-next-hello-world.git
-cd loopback-next-hello-world
+This example will explain:
+
+* the structure of a Loopback.next application
+* how to add a custom component (TK)
+* how add a custom sequence (TK)
+
+When you have this example installed and running, you will be able to make a request to the server with a name and get back a HTTP 200 response: 
+
+[TODO: add screenshot of output here]
+
+## <a name="setup"></a>Setup 
+
+1. Install `node`, `npm`, and `TypeScript` 
+
+	You should have `node` (>= 7.6.0), `npm` (>= 3.10), and `TypeScript`  (>= 2.0.0) installed. 
+	
+	For the latest information on project dependencies for LoopBack, please check this link: 
+[https://github.com/strongloop/loopback-next/wiki/Installation](https://github.com/strongloop/loopback-next/wiki/Installation).
+
+2. Clone this repo: 
+	
+	`git clone https://github.com/strongloop/loopback-next-hello-world.git`
+
+3. Switch to your new directory: 
+	
+	`cd loopback-next-hello-world`
+
+4. Install your dependencies: 
+
+	```
 npm install -S @loopback/core
```

Comment:
Once merged @emckean will work on README.  I've communicated this point with @emckean .